WEB_SERVICE/WEB/_LOW/SSL_RACCOON_ATTACK/meta.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "Raccoon Attack on SSL/TLS",
+  "short_description": "A timing vulnerability in the Diffie-Hellman key exchange that allows attackers to recover sensitive data through precise measurement of server response times.",
+  "risk_rating": "low",
+  "references": {
+    "Raccoon Attack Research Paper": "https://raccoon-attack.com"
+  },
+  "privacy_issue": true,
+  "security_issue": true,
+  "categories": {
```

Comment:
i think you need to add these: 
```python
"OWASP_MASVS_L1": [
      "MSTG_CRYPTO_4"
    ],
    "OWASP_MASVS_L2": [
      "MSTG_CRYPTO_4"
    ],
    "OWASP_MASVS_v2_1": [
      "MASVS_CRYPTO_1",
      "MASVS_CRYPTO_2"
    ],
```
![image](https://github.com/user-attachments/assets/ae71dbe3-efb5-4702-9a37-1a6643266853)